addons/l10n_eu_oss/models/res_company.py

```diff
@@ -82,8 +82,17 @@ def _map_eu_taxes(self):
                                     }).id,
                                     'noupdate': True,
                                 })
+                            foreign_tax_name = f'{tax_amount}% {destination_country.code} {destination_country.vat_label}'
+                            existing_foreign_taxes = self.env['account.tax'].search([
+                                ('company_id', 'child_of', company.root_id.id),
+                                ('name', 'like', foreign_tax_name),
+                                ('type_tax_use', '=', 'sale'),
+                                ('country_id', '=', company.account_fiscal_country_id.id),
+                            ], order='sequence,id desc', limit=1,
+                            )
+                            foreign_tax_copy_name = existing_foreign_taxes and '%s %s' % (existing_foreign_taxes.name, _("(Copy)"))
```

Comment:
```suggestion
                            foreign_tax_copy_name = existing_foreign_taxes and _('%(tax_name)s (Copy)', tax_name=existing_foreign_taxes.name)
```
Slightly easier for the [translation](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#use-translation-method-correctly), it gives (a very little bit of) context.

Reply:
Well I'm fine with the changes but i just checked and `l10n_eu_oss`is not on Transifex sooo I'll add the translations myself, making it easier for the translators actually didn't change much :joy: :

Reply:
Indeed, but it's still a good practice I just wanted to share, and it might help if someone needs to update the translations manually in the future :nerd_face: